test/nidmExampleDataTest.m

```diff
@@ -1,10 +1,46 @@
+%To run the below tests use the following code:
```

Comment:
Can you create a script inside the `test` folder including these three lines of code.